android/sdk/src/main/java/com/taobao/weex/utils/WXSoInstallMgrSdk.java

```diff
@@ -242,6 +268,22 @@ public static void copyStartUpSo() {
         } else {
           WXEnvironment.extractSo();
         }
+
+        Closeable w = null;
+        try {
+          if(!versionFile.exists())
+            versionFile.createNewFile();
+          FileWriter fileWriter = new FileWriter(versionFile);
```

Comment:
Move `fileWriter` out of the try clause and close the `fileWriter` in finally would be better.